src/main/webapp/resources/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.navbar {
+  width: 100%;
+  max-width: 1400px;
+  flex-flow: row nowrap;
+}
+
+.form-line {
+  max-width: 60vw;
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  flex-grow: 1;
+  margin: 0;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  background-color: #a9a9a9;
+  width: 100%;
+}
+
+#footer {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  min-height: 30px;
+  height: 5vh;
+}
+
+#links {
+  text-transform: uppercase;
+}
+
+.link {
+  color: black;
+}
+
+.link:hover {
+  color: black;
+}
+
+#home-page {
+  align-items: center;
+  justify-content: center;
+}
+
+#welcome {
+  margin: 30px;
+}
+
+#searchbar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+}
+
+.buttonline {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.btn {
```

Comment:
Be consistent, it's either .button or .btn

Reply:
The class `btn` is from Bootstrap.
For consistency I renamed `button-line` class to `btn-line`